micromasters/urls.py

```diff
@@ -52,6 +53,8 @@
         name='financial_aid_action'),
     url(r'^api/v0/financial_aid_course_price/(?P<program_id>[\d]+)/$',
         GetLearnerPriceForCourseView.as_view(), name='financial_aid_course_price'),
+    url(r'^api/v0/financial_aid_documents_sent/(?P<financial_aid_id>[\d]+)/$',
+        DocumentsSentView.as_view(), name='financial_aid_documents_sent'),
```

Comment:
in short, i'm asking why the url isn't `api/v0/financial_aid/(?P<financial_aid_id>[\d]+)/$`, and the view isn't `FinancialAidDetailView` or something like that. adding a brand new url path and view for every action we want to perform is not a good pattern. if we're at least considering REST principles, we should be thinking about this in terms of the resource, which is `FinancialAid`. in essence, we're adding this functionality to be able to update (ie: `PUT`) a `FinancialAid` record, identified by id. do we need to follow REST principles exactly? no, but this seems like a pretty simple case to me. with that said, if there's not (a) a clear and easy path forward to implement what i'm describing, nor (b) a future need to update `FinancialAid` records via API, i'm ok with leaving this as-is due to time constraints.